components/script/dom/idbtransaction.rs

```diff
@@ -229,6 +229,7 @@ impl IDBTransactionMethods<crate::DomTypeHolder> for IDBTransaction {
         // returns the same IDBObjectStore instance.
         let mut store_handles = self.store_handles.borrow_mut();
         let store = store_handles.entry(name.to_string()).or_insert_with(|| {
+            // get key path
```

Comment:
I don't know what this comment means.

Reply:
Fixed the comment, it was meant to be a todo.